bulk_handlers.go

```diff
@@ -33,7 +33,14 @@ func BulkCreate(c echo.Context) error {
 		return fmt.Errorf("failed to pull identity from request")
 	}
 
-	user := &m.User{TenantID: tenantID, UserID: id.Identity.User.UserID}
+	userID, ok := c.Get(h.USERID).(int64)
+
+	if !ok {
+		c.Logger().Warnf("failed to pull user id from request")
+		return fmt.Errorf("failed to pull user id from request")
+	}
```

Comment:
I think we might receive bulk create requests without the user ID

Reply:
yes that's right when we authorise with PSK we don't have userid

Reply:
I did the change to last commit.